src/rpcserver.h

```diff
@@ -114,6 +114,12 @@ class CRPCTable
     std::string help(std::string name) const;
 
     /**
+     * add or replace a CRPCCommand to the dispatch table
+     * @param command rpc command to add or replace
+     */ 
+    void AddOrReplaceCommand(const CRPCCommand command);
```

Comment:
You are totally right. It must be `const CRPCCommand& command` (passing pointers). I just pulled in your commit.
I'm also aware that this pointer-passing-approche is not intuitive. But i'm still trying to keep the changeset-size at minimum. I also think a rpc calls will be added when the app starts (modules get loaded) then they stay alive until the app detects a shutdown so the lifetime management tend to be not so complex.